custom/ilsgateway/models.py

```diff
@@ -432,3 +432,12 @@ class RequisitionReport(models.Model):
     location_id = models.CharField(max_length=100, db_index=True)
     submitted = models.BooleanField(default=False)
     report_date = models.DateTimeField(default=datetime.utcnow)
+
+
+class SupervisionDocument(models.Model):
+    document = models.FileField(upload_to='supervision_documents/')
```

Comment:
You can't store anything on the filesystem since you're never guaranteed to arrive at the same worker machine when you try and access it.

I would recommend either storing it in postgres: https://wiki.postgresql.org/wiki/BinaryFilesInDB#What_is_the_best_way_to_store_the_files_in_the_Database.3F or in the `commcarehq__meta` couch db as an attachment. (meta is better in case you need to add a couch view to index the documents).